src/lib/colourType.js

```diff
@@ -12,5 +12,5 @@ export const isHex = colour => {
 export const isRGBorHSL = colour => /^(rgb|hsl)a?\(.*?\)/.test(colour);
 
 export const isKeyword = colour => {
-    return ~Object.keys(colourNames).indexOf(colour.toLowerCase());
```

Comment:
You can also use https://npmjs.com/has if you want an impossible-to-break version of `Object.prototype.hasOwnProperty.call`.